tests/queries/0_stateless/01546_log_queries_min_query_duration_ms.sql

```diff
@@ -0,0 +1,10 @@
+set log_queries=1;
+set log_queries_min_query_duration_ms=1;
+
+select '01546_log_queries_min_query_duration_ms-fast', * from system.one format Null;
```

Comment:
Query can still take more than one milliseconds (and even more than one minute), let's remove this case at all.

Reply:
1ms yes, but 100ms should not, I get it back (hope you are not against) to cover the case that QueryStart is not logged if log_queries_min_query_duration_ms wasn't triggered (since there is a question about this already - https://github.com/ClickHouse/ClickHouse/pull/16529#issuecomment-719329771)

Reply:
Changed the delay to 300ms (just in case) in test